app/services/ems_cloud_dashboard_service.rb

```diff
@@ -15,9 +15,10 @@ def aggregate_status_data
 
   def aggregate_status
     {
-      :quadicon => @controller.instance_exec(@ems, &EmsInfraDashboardService.quadicon_calc),
-      :status   => status_data,
-      :attrData => attributes_data,
+      :quadicon      => @controller.instance_exec(@ems, &EmsInfraDashboardService.quadicon_calc),
+      :status        => status_data,
+      :refreshStatus => refresh_data,
```

Comment:
```suggestion
      :refresh_status => refresh_data,
```

Again not sure on javascript vs Ruby conventions here.

Reply:
@fryguy it is working, in fact several other places are using camelcase in ruby files to get data in react( Wg: https://github.com/ManageIQ/manageiq-ui-classic/blob/634700500a5aa8586f08dc6470dddb23fb551ca6/app/services/ems_infra_dashboard_service.rb#L34) , let me know if this is not recommended, will change it.

Reply:
It's fine, just confusing for me since I'm not used to the react side of things...as long as we are consistent I'm fine with it.